src/core/dbus.c

```diff
@@ -1101,16 +1101,16 @@ int bus_fdset_add_all(Manager *m, FDSet *fds) {
         return 0;
 }
 
-int bus_foreach_bus(
+int bus_foreach_bus_signal(
```

Comment:
Not a matter, but bus_foreach_bus_send_signal() or bus_foreach_send_signal() is better?